### PR TITLE
Add Biography

### PR DESCRIPTION
**Pull request by bohanyin**
For hw01, I added my biography using several elements in markdown, including emphasis, lists, headers, an image, and a link.  It is user-friendly.

### CODE REVIEW

**Review by Luorao — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |   x        |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |    x       |              |            |

Good job!  Well-organized code.
Finished all the tasks within a single commit, nice!
Good luck with finding a job :D

**Comment by mengdihuang** [comment]
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Your bio is very clear and organized!
You completed all the requirements.
I also love cooking so maybe we can become friends haha:)

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+# hw01
+
+# Bohan Yin's Biography
+
+Hi, my name is **Bohan Yin**, a second-year MPP student from Uchicago.  I am interested in data science and international relations (*although they seem unrelated*). During my spare time, I do a lot of things:
+> *Playing Tennis
+> Cooking and Baking
+> Playing video games
+> etc...*
+
```
Comment (rrodriguezbarron) [inline README.md:10]: This looks nice. But how about trying to make a list? That would also work and it would look better in the final version IMHO.

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |             |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Good use of emphasis
* The image loaded correctly
* I would recommend practicing the markdown for lists - they may come in handy in future assignments

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+I am currently seeking for a full-time job. Click my [LinkdIn](www.linkedin.com/in/bohan-yin) to know more about me.
```
Comment (boruis12) [inline README.md:11]: small typo- "Linkedin"

**Review by boruis12 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:
Great use of header and emphasis! Your link and image loaded properly. 
Regarding the hobby list, I am guessing that your are trying to make a list in italics. You may want to use separate **  for each individual item in your list for that to work, your way of doing it looks pretty cool though. I did not know list and ** combined together can generate a nice way of highlighting the text. Nice work!

**Pull request closed without merging**